Sources/SuperbDemo/GitHub+Providers.swift

```diff
@@ -19,3 +19,11 @@ extension GitHubOAuthProvider {
     )
   }
 }
+
+extension TwitterOAuthProvider {
```

Comment:
Should this be in a file called `GitHub+Providers`?

Reply:
Oh good catch